src/components/ContentSidebar/DetailsSidebar.js

```diff
@@ -17,32 +17,37 @@ import SidebarFileProperties from './SidebarFileProperties';
 
 import './DetailsSidebar.scss';
 
+type ExternalProps = {
```

Comment:
@JustinHoldstock name ideas?

Reply:
I actually like the current name! We always assume that `Props` is an internal type, whereas External explains something different than what we are expecting. Do you want to export that, or is it not longer referenced outside of here?